src/index.jsx

```diff
@@ -1,26 +1,28 @@
-import 'babel-polyfill';
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
 
+import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
+import Header, { messages as headerMessages } from '@edx/frontend-component-header';
 import {
-  APP_INIT_ERROR, APP_READY, subscribe, initialize,
+  APP_INIT_ERROR, APP_READY, initialize, subscribe,
 } from '@edx/frontend-platform';
 import { AppProvider, ErrorPage } from '@edx/frontend-platform/react';
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import Header, { messages as headerMessages } from '@edx/frontend-component-header';
-import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
 
+import './assets/favicon.ico';
+import { DiscussionsHomeContainer } from './discussions';
 import appMessages from './i18n';
-import ExamplePage from './example/ExamplePage';
-
 import './index.scss';
-import './assets/favicon.ico';
+import store from './store';
 
 subscribe(APP_READY, () => {
   ReactDOM.render(
-    <AppProvider>
+    <AppProvider store={store}>
```

Comment:
I see in console this error:

![Screenshot_2020-07-14_12-53-27](https://user-images.githubusercontent.com/18251194/87411899-0ae77880-c5d1-11ea-83c1-40dffb79751b.png)

Probably you need to wrap this into `IntlProvider`?

Reply:
AppProvider already does that: https://github.com/edx/frontend-platform/blob/master/src/react/AppProvider.jsx